content/learn/quick-start/getting-started/setup.md

```diff
@@ -143,84 +143,137 @@ You might think to simply develop in release mode instead, but we recommend agai
 
 ### Enable Fast Compiles (Optional)
 
-Bevy can be built just fine using default configuration on stable Rust. However for maximally fast iterative compiles, we recommend the following configuration:
+Bevy can be built just fine using default configuration on stable Rust.
+Unfortunately, the compile times are rather long.
+This section explains how to speed up iterative compiles: the amount of time it takes to rebuild your project after changing a single file.
 
-* **Enable Bevy's Dynamic Linking Feature**: This is the most impactful compilation time decrease! If `bevy` is a dependency, you can compile the binary with the "dynamic_linking" feature flag (enables dynamic linking). **Important!** On Windows you _must_ also enable the [performance optimizations](#compile-with-performance-optimizations) or you will get a [`too many exported symbols`](https://github.com/bevyengine/bevy/issues/1110#issuecomment-1312926923) error.
+#### Dynamic Linking
+
+This is the most impactful compilation time decrease!
+You can compile `bevy` as dynamic library, preventing it from having to be statically linked each time you rebuild your project. You can enable this with the `dynamic_linking` feature flag.
+
+```sh
+cargo run --features bevy/dynamic_linking
+```
+
+If you don't want to add the `--features bevy/dynamic_linking` to each run, this flag can permanently be set with this command (edits `Cargo.toml` for you):
+
+```sh
+cargo add bevy -F dynamic_linking
+```
+
+{% callout(type="note") %}
```

Comment:
```suggestion
{% callout(type="warning") %}
```

Making this a warning instead of a note calls out how significant this is, and also nicely contrasts the note right beneath it.